### PR TITLE
Fix sample code in `README.md`

### LINKED ISSUE #19: `README.md` uses undefined method in sample code
The `README.md` gives a sample code illustrating how elektra could be used. One of the used methods is `Value` which is expected to return the value of a `Key`. 
The `Key` does not provide such method and I have not seen any other method to fetch the value of a key. 
We should introduce a `Value` method on the `Key` type.

### PR DESCRIPTION
**Pull request by tmakar**
fixes #19

### CODE REVIEW

File: kdb/key.go [hunk 1]
```diff
@@ -223,7 +223,7 @@ func (k *CKey) Bytes() []byte {
 }
 
 // String returns the string value of the Key.
-func (k *CKey) String() string {
+func (k *CKey) Value() string {
```
Comment (kodebach) [inline kdb/key.go]: I don't think it's a good idea. As you can see, it calls `keyString`. There is also `keyValue`, do what you propose here could be exceedingly confusing.

```suggestion
func (k *CKey) StringValue() string {
```

or 

```suggestion
func (k *CKey) ValueString() string {
```

would be acceptable IMO, however, as stated in #19 I think we should wait for ElektraInitiative/libelektra#4853 before changing the Go API.

**Review by kodebach — CHANGES_REQUESTED** [review]

File: kdb/key.go (hunk 1 above)
Comment (tmakar) [reply kdb/key.go]: ok, I will then revert this change.

See comment: https://github.com/ElektraInitiative/go-elektra/issues/19#issuecomment-1464892449

**Comment by markus2330** [comment]
Please let first review from two others.

**Review by kodebach — APPROVED** [review]
LGTM

**Review by hannes99 — APPROVED** [review]

**Comment by hannes99** [comment]
Looks good 👍

**Comment by tmakar** [comment]
@markus2330 This is now ready to be merged. What do you think?

File: README.md [hunk 2]
```diff
@@ -73,33 +73,45 @@ package main
...
+
 	keyName := "/go/elektra"
 
 	handle := kdb.New()
 
 	// Open the handle, this is a separate step since there can be different implementations of the KDB interface.
-	_ = handle.Open()
+	err := handle.Open()
+	if err != nil {
+		fmt.Println("Errow while opening database")
```
Comment (markus2330) [inline README.md]: Can you also print the error?

```suggestion
		fmt.Println("Error while opening the database")
```

File: README.md [hunk 3]
```diff
@@ -73,33 +73,45 @@ package main
...
+	if err != nil {
+		fmt.Println("Error while creating new key", err)
+		os.Exit(1)
+	}
 	defer parentKey.Close()
 
-	_, _ = handle.Get(ks, parentKey)
+	_, err = handle.Get(ks, parentKey)
+	if err != nil {
+		fmt.Println("Error while retrieving parent key and all keys beneath")
```
Comment (markus2330) [inline README.md]:
```suggestion
		fmt.Println("Error while getting parent key and all keys below")
```

**Review by markus2330 — APPROVED** [review]
Great start, looking forward to more PRs! Please do the little fixes, then @atmaxinger can merge.

**Comment by tmakar** [comment]
@atmaxinger Can be merged!

**Pull request merged**